setup.py

```diff
@@ -39,7 +39,8 @@
                    'Topic :: Utilities'],
       python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
       install_requires=[
-          'Django>=1.11,<3.0',
+          'Django>=1.11,<4.0',
           'requests',
+          'six'
```

Comment:
I'm fine either way, but an alternative would be to follow Django's lead and drop Python 2 support as was done in #20. If that is still not desired at this time, this change is very useful for testing future compatibility.

Reply:
I'm happy to drop Py2. Is there any chance of getting this merged (with / without Py2) - as Django 3 is imminent, and this is currently a blocker for it.